usage/www/index.html

```diff
@@ -2,19 +2,7 @@
   <body style="padding: 0; margin: 0;">
     <div style="position: absolute; top: 5px; right: 5px;">
       <select>
-        <option>Geometry/Picking</option>
-        <option>Geometry/OBJViewer</option>
-        <option>Geometry/PointCloud</option>
-        <option>Geometry/PolyDataViewer</option>
-        <option>Geometry/PolyDataWithData</option>
-        <option>Geometry/ProcessingPipeline</option>
-        <option>Geometry/SourceViewer</option>
-        <option>Geometry/Glyph</option>
-        <option>Geometry/CutterExample</option>
         <option>Volume/SliceRendering</option>
-        <option>Volume/SyntheticVolumeRendering</option>
-        <option>Volume/VolumeRendering</option>
-        <option>Volume/DynamicUpdate</option>
```

Comment:
Is this change correct? Looks like you have removed these examples for testing, but forgot to re-enable them before commit?